src/components/Selector.tsx

```diff
@@ -153,9 +153,10 @@ const customStyles: Styles = {
     display: 'inline-block'
   }),
   // Allow the valueComponent to handle it's own padding when present.
+  // If input is present in the field, it takes up 4px.
   valueContainer: (styles, state) => ({
     ...styles,
-    ...(state && state.hasValue && { paddingLeft: 0 })
+    paddingLeft: state.hasValue ? '6px' : '10px'
```

Comment:
Can you explain this change?

Reply:
Basically, selectors already have padding in them that they use for spacing of the searchable field's blinking cursor & text elements when searching. We were overriding this, resulting in the cursor being in a weird spot. old: 
![image](https://user-images.githubusercontent.com/29407814/107685884-65c17b00-6c59-11eb-9cc6-bec5b7e127dc.png)

new:
![image](https://user-images.githubusercontent.com/29407814/107698083-e0de5d80-6c68-11eb-889b-ae8f6fbe5d1d.png)